sql/moz-fx-data-marketing-prod/google_search_console_derived/url_impression_exports_empty_check_v1/metadata.yaml

```diff
@@ -0,0 +1,16 @@
+friendly_name: URL Impression Exports Empty Check
+description: |-
+  Raises an error if any of the Google Search Console URL impression exports are missing data for the given date.
```

Comment:
Is there a reason why this is not done with ETL checks?

Reply:
See [other](https://github.com/mozilla/bigquery-etl/pull/4892#discussion_r1468022013) [threads](https://github.com/mozilla/bigquery-etl/pull/4892#discussion_r1482004593).